app/src/main/java/com/appunite/loudius/ui/pullrequests/PullRequestsScreen.kt

```diff
@@ -284,6 +287,7 @@ fun PullRequestsScreenEmptyListPreview() {
 }
 
 @Preview("Pull requests - Loading")
+@ShowkaseComposable(skip = true)
 @Composable
```

Comment:
Nitpick: If you will change something in codebase, you can change the order here😅
```
@Preview("Pull requests - Loading")
@Composable
@ShowkaseComposable(skip = true)
```